asv_bench/benchmarks/rolling.py

```diff
@@ -25,6 +25,25 @@ def peakmem_rolling(self, constructor, window, dtype, method):
         getattr(self.roll, method)()
 
 
+class Apply:
+    params = (
+        ["DataFrame", "Series"],
```

Comment:
any reason not to just put `(pd.DataFrame, pd.Series)` and avoid the `getattr`?

Reply:
It was the nicest way to have it label nicely in the `params` and not be an `if/else` in the `setup`. Plus the `getattr` is in the `setup` so it won't impact the benchmark time.